nowcasting_utils/metrics/validation.py

```diff
@@ -9,7 +10,12 @@
 
 
 def make_validation_results(
-    predictions, truths, gsp_ids, t0_datetimes_utc, batch_idx: Optional[int] = None
+    predictions_mw,
+    truths_mw,
+    gsp_ids: List[int],
+    t0_datetimes_utc: pd.DatetimeIndex,
+    batch_idx: Optional[int] = None,
+    forecast_sample_period: timedelta = timedelta(minutes=30),
```

Comment:
Regarding the docstring:

Please update the description of the columns that are made

Also, please add docstring info about the expected shape of `predictions_mw` and `truths_mw`.